packages/utils/src/lib/sort-report.integration.test.ts

```diff
@@ -0,0 +1,52 @@
+import { REPORT_MOCK } from '@code-pushup/testing-utils';
+import { scoreReport } from './scoring';
+import { sortReport } from './sort-report';
+
+describe('sortReport', () => {
```

Comment:
I am missing information of the sorting priorities. E.g. `should sort the audits and groups based on X, Y then Z and categories based on A then B`

And then for each section I would reiterate what the person should look for in the results, e.g.
`Categories (sorted based on X then Y):`